smdebug/tensorflow/tensor_ref.py

```diff
@@ -96,8 +99,14 @@ def from_variable(cls, variable, export_name=None, mode=None, original_tensor=No
                 # for mirrored variable value this will be the mirrored variable
                 original_tensor = variable
 
+            if is_tf_version_2x() and tf.executing_eagerly() and isinstance(variable, tf.Variable):
```

Comment:
and eagerly executing? 
Rest of the above you are doing things when not executing eagerly and version > 2.x 

Can you make a list - 
what will be saved for - 
> 2.0 and not eager
> 2.0 and eager
< 2.0

Reply:
Can you point me to test cases for these 3 cases.

Reply:
2.0 and eager -> this if condition is the default case. executed by all tests in tests/tensorflow2.
2.0 and not eager = 1.x default behavior -> executed by all tests in tests/tensorflow for 1.x and all tests in non-eager mode in tests/tensorflow2 for 2.x.

I will execute test_keras_fit() in all 3 cases (With vanilla TF) and paste the output as gist here

Reply:
![image](https://user-images.githubusercontent.com/9017059/76253151-5a72fd80-6207-11ea-9552-356d86ccd63d.png)

Reply:
Thx for posting with results, is it result of save_all=True. Why don't i see gradients for 1.15 ?

Reply:
Nice thanx for the list